lib/eldap/src/eldap.erl

```diff
@@ -1088,6 +1093,9 @@ v_bool(true)  -> true;
 v_bool(false) -> false;
 v_bool(_Bool) -> throw({error,concat(["not Boolean: ",_Bool])}).
 
+v_size_limit(I) when is_integer(I), I>=0 -> I;
+v_size_limit(_I) -> throw({error,concat(["size_limit not positive integer: ",_I])}).
```

Comment:
`=:= 0` is non-negative, not positive 😄 (I see there's an error just like it, below your change)

Reply:
So update the error message, or the actual check evaluation, seems like we are enforcing the number being greater than 0, so must be a positive integer. Didn't really understand what you mean 😅

Reply:
I was stating that since you're checking if something is bigger or equal to 0 the "correct" message to throw in case it isn't should be "size_limit not non-negative integer" or better still "size_limit negative integer", but it's a minor thing (it would make it incompatible with other messages outside of the scope of the pull request, though).